### PR TITLE
Add new layout

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [ ] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds 2 new layouts: "Center" and "Topper with 2 articles"
This is in order to get a bit more diversity with our layouts.
More layouts to follow.

Screenshot 2022-01-26 at 16 36 12

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Create a new Newsletter
3. Insert the new layouts and send the newsletter

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-newspack-newsletters-layouts.php [hunk 1]
```diff
@@ -120,12 +120,35 @@ public static function register_meta() {
...
 	public static function layout_token_replacement( $content, $extra = [] ) {
+		$home_url       = get_home_url();
 		$sitename       = get_bloginfo( 'name' );
 		$custom_logo_id = get_theme_mod( 'custom_logo' );
 		$logo           = $custom_logo_id ? wp_get_attachment_image_src( $custom_logo_id, 'medium' )[0] : null;
+		$date           = gmdate( get_option( 'date_format' ) );
+		$bg_color       = '#ffffff';
+		$text_color     = '#000000';
+
+		if ( function_exists( 'newspack_setup' ) ) {
```
Comment (adekbadek) [inline includes/class-newspack-newsletters-layouts.php:131]: Does that serve the purpose of determining if a Newspack theme is in use? If so, that's not readily understandable, there should be a comment here.

**Review by adekbadek — COMMENTED** [review]
Left one nitpicky thing.

I do see layout issues with layout previews, but with the new layouts UI (no scrollable preview) these won't be visible. Could you please merge in `master`, so this can be confirmed?

image

image

**Comment by thomasguillot** [comment]
> I do see layout issues with layout previews, but with the new layouts UI (no scrollable preview) these won't be visible. Could you please merge in `master`, so this can be confirmed?

Done, alongside https://github.com/Automattic/newspack-newsletters/pull/709/commits/0868281a4d3cc73a63c79fc85e26e6b0d684b786

File: includes/class-newspack-newsletters-layouts.php (hunk 1 above)
Comment (thomasguillot) [reply includes/class-newspack-newsletters-layouts.php:132]: fixed: https://github.com/Automattic/newspack-newsletters/pull/709/commits/e3fd6558b5107efc475d99bcc9688ce8165a0fd8

**Comment by adekbadek** [comment]
Noting that this PR should be updated – there are [commits here which are already on master](https://github.com/Automattic/newspack-newsletters/pull/709#commits-pushed-0088261), which makes it hard to review.

**Comment by thomasguillot** [comment]
Putting this on hold. Layouts are more likely to change once https://github.com/Automattic/newspack-newsletters/pull/679 is merged.

**Pull request closed without merging**